src/main/java/seedu/address/logic/commands/DeleteNoteCommand.java

```diff
@@ -0,0 +1,109 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDICES;
```

Comment:
Just a note Axel - sorry about this, I think I deleted the PREFIX_INDICES in my previous PR (because I've changed how Mark Command functions) - in case it causes some issues during merging.